index.html

```diff
@@ -289,7 +289,114 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+       <div id="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form  id ="form" action="https://formspree.io/f/mwkakrqj" method="GET">
+          <input
+            type="text"
+            name="fullname"
+            id="fullname"
+            placeholder="Full name"
+            required
+          />
+          <div class="name-form">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              placeholder="First name"
+              required
+            />
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              placeholder="Last name"
+              maxlength="30"
+              required
+            />
+          </div>
+          <input
+            type="email"
+            id="email"
+            name="_replyto"
+            placeholder="Email address"
+            required
+          />
+          <textarea
+            name="message"
+            placeholder="Enter text here"
+            required
+          ></textarea>
+          <button type="submit" class="link centered-link">Send</button>
```

Comment:
Well done for the button hover effects! I like that it changes the color smoothly when I hover the mouse over the button. But the text, border color, and size needs to be the same as it is in the Figma design, so make sure to update it.
![image](https://user-images.githubusercontent.com/34779928/190404125-56700f88-0d8a-49e0-b677-3b71c4e2c209.png)